pkg/kubectl/kubectl.go

```diff
@@ -22,6 +23,20 @@ func GetClientset() (*rest.Config, *dynamic.DynamicClient, error) {
 	return config, cs, err
 }
 
+func GetClientset() (*rest.Config, *kubernetes.Clientset, error) {
```

Comment:
whats the difference between dynamic and non dynamic?

Reply:
dynamic is for custom resources 
regular is for corev1